libs/designsystem/package.json

```diff
@@ -3,10 +3,11 @@
   "version": "0.0.0",
   "dependencies": {
     "@angular/cdk": "8.2.3",
+    "@fontsource/roboto": "4.2.1",
     "@ionic/angular": "5.5.2",
     "highcharts": "8.0.4",
     "moment": "2.24.0",
-    "@fontsource/roboto": "4.2.1"
+    "tslib": "^2.0.0"
```

Comment:
I belive this should be a devDependency

Reply:
I think you're right

Reply:
https://github.com/kirbydesign/designsystem/pull/1401/files/73c6c835f4114e93ca452739607a7b6c82f27d17#r595833123